docs/blog/v8.md

```diff
@@ -0,0 +1,135 @@
+---
+image: https://images.unsplash.com/photo-1485795046599-702122cd1267?crop=entropy&cs=tinysrgb&fit=crop&fm=jpg&h=500&ixid=eyJhcHBfaWQiOjF9&ixlib=rb-1.2.1&q=80&w=1950
+title: Announcing "auto" 8.0.0
+author:
+  name: Andrew Lisowski
+  url: https://twitter.com/HipsterSmoothie
+  email: lisowski54@gmail.com
+---
+
+We are excited to announce the release of `auto` 8.0.0.
+This release brings a new command `next`, a new `shipit` use, more powerful label configuration, and more,
+
+If you don't know what `auto` yet, it's a CLI tool helps you version your code and automate any part of your release.
```

Comment:
```suggestion
If you are unfamiliar, `auto` is a CLI tool helps you version your code and automate any part of your release pipeline.
```

Reply:
"is a CLI tool **which**" or is it **that**?